railties/lib/rails/generators/app_base.rb

```diff
@@ -92,7 +92,7 @@ def self.add_shared_options_for(name)
                                            desc: "Set up the #{name} with Gemfile pointing to your Rails checkout"
 
         class_option :edge,                type: :boolean, default: nil,
-                                           desc: "Set up the #{name} with Gemfile pointing to Rails repository"
+                                           desc: "Set up the #{name} with a Gemfile pointing to the 7-1-stable branch on the Rails repository"
```

Comment:
Can we reuse the logic that generates the branch in the gemfile to also generate the description message?